test/forces.jl

```diff
@@ -25,7 +30,7 @@ include("testcases.jl")
 
     diff_findiff = -(E2 - E1) / ε
     diff_forces = dot(F1[1][1], disp)
-    @test abs(diff_findiff - diff_forces) < 2e-6
+    @test abs(diff_findiff - diff_forces) < 1e-4
```

Comment:
1e-4 is really big. Do you think this than still tests anything?

Reply:
really? 1e-4 is not that big... Usually I just take twice what I compute or something, but then very quickly we have to adjust when we change things (like here when I added a kpoint). Having larger tolerances means we don't have to worry about it so much

Reply:
I don't like this strategy. So far it surely was annoying to tweak tolerances every now and then, but each time I learned a lot about details of our code or of a particular feature (like forces computation). Also you don't miss unforseen influences of e.g. SCF convergence parameters so easily. I really would for keeping tolerances as tight as possible at the expense to tweak them when needed.

Reply:
I think it really depends. We should definitely have a few core tests with tight tolerance criteria to catch these sort of things. But here we just want to see if the formulas for implementing the forces are correct, so we shouldn't really focus on a very tight convergence criterion